pyaedt/modules/AdvancedPostProcessing.py

```diff
@@ -395,6 +397,8 @@ def plot_field_from_fieldplot(
             model.camera_position = view
         elif view != "isometric":
             self.logger.warning("Wrong view setup. It has to be one of xy, xz, yz, isometric.")
+        if is_pcb:
+            model.z_scale = 5
```

Comment:
@maxcapodi78 Should it be a parameter or will it always work with this value for the `z_scale` property?

Reply:
Ideally could be a parameter